icu4c/source/common/uresbund.cpp

```diff
@@ -1444,10 +1446,10 @@ U_CAPI UResType U_EXPORT2 ures_getType(const UResourceBundle *resB) {
 }
 
 U_CAPI const char * U_EXPORT2 ures_getKey(const UResourceBundle *resB) {
+  // TODO: Trace ures_getKey?
```

Comment:
I guess not usually.

We usually get the key string to decide whether we want the value, or to make a key-value pair. Tracing the value should suffice.

However, I believe we have some data (e.g., in res_index) where the key strings *are* the data. Tracing the enclosing table should suffice.

You could leave it as is now.
Or you could add a separate trace event for getKey().